common/changes/@microsoft/rush/dotenv_2025-03-10-22-13.json

```diff
@@ -0,0 +1,10 @@
+{
+  "changes": [
+    {
```

Comment:
Is `<repo root>/.env` intended to be committed to Git? Or is it for the user only?  We should document the intended usage.